src/main/webapp/feed.html

```diff
@@ -9,16 +9,12 @@
     </head>
 
     <body onload="buildUI()">
-    <nav>
-      <ul id="navigation">
-        <li><a href="/">Home</a></li>
-        <li><a href="/aboutus.html">About Our Team</a></li>
-        <li><a href="/community.html">Community</a></li>
-        <li><a href="/feed.html">Feed</a></li>
-        <li><a href="/stats.html">Stats</a></li>
-        <li><a href="/user-page.html">User Page</a></li>
-      </ul>
-    </nav>
+        <nav>
+            <ul id="navigation">
+              <li><a href="/">Home</a></li>
+              <li><a href="/aboutus.html">About Our Team</a></li>
+            </ul>
+        </nav>
```

Comment:
this file is behind the one that's in the master. Would be best to remove this commit